### PR TITLE
Add MFA classes and calls

### PR DESCRIPTION
**Pull request by pliao28**

### CODE REVIEW

File: src/mfa/mfa.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { WorkOS } from '../workos';
+import { ChallengeFactorOptions } from './interfaces/challenge-factor-options';
+import { Challenge } from './interfaces/challenge.interface';
+import { EnrollFactorOptions } from './interfaces/enroll-factor-options';
+import { Factor } from './interfaces/factor.interface';
+import { VerifyFactorOptions } from './interfaces/verify-factor-options';
+import { VerifyResponse } from './interfaces/verify-factor-response';
+
+export class MFA {
```
Comment (maxdeviant) [inline src/mfa/mfa.ts:9]: nit: Can we change the casing to be `Mfa`?

```suggestion
export class Mfa {
```

**Comment by robframpton** [comment]
@pliao28 this should be good to go once we fix the snapshot tests. Let's also bump the version in package.json to a prerelease number. 

We'll also have to make a small change to the Semaphore publish job to add the `--tag` flag to the `npm publish` command. Something like `npm publish --tag=beta`.

**Comment by robframpton** [comment]
@pliao28 I updated the publish command so once the CI jobs have passed we should be good to publish the first alpha version in Semaphore.

Installing the alpha version will be done like so.

```
yarn add @workos-inc/node@alpha
```

**Comment by pliao28** [comment]
@robframpton  preview published

**Review by robframpton — APPROVED** [review]

File: src/workos.ts [hunk 2]
```diff
@@ -20,8 +20,9 @@ import { Passwordless } from './passwordless/passwordless';
 import { Portal } from './portal/portal';
 import { SSO } from './sso/sso';
 import { Webhooks } from './webhooks/webhooks';
+import { Mfa } from './mfa/mfa';
 
-const VERSION = '2.4.0';
+const VERSION = '2.5.0-alpha.1';
```
Comment (pliao28) [inline src/workos.ts:25]:
```suggestion
const VERSION = '2.4.0';
```

**Comment by pliao28** [comment]
@robframpton  reverted version back to 2.4.0 off of the release version 2.5.0-alpha.1 , Could you merge and publish, I'm not authorized for the base branch

**Pull request merged**